app/views/spree/admin/overview/_enterprises.html.haml

```diff
@@ -1,11 +1,64 @@
+      %span.one.columns.omega
+        %span.icon-remove-sign
+    %a.sixteen.columns.alpha.button.bottom.red{ href: "#{main_app.new_admin_enterprise_path}" }
+      = t "spree_admin_enterprises_none_create_a_new_enterprise"
+      %span.icon-arrow-right
 
   - else
-    = render 'enterprises_tabs'
-    = render 'enterprises_hubs_tab'
-    = render 'enterprises_producers_tab'
-    = render 'enterprises_footer'
+    %div.sixteen.columns.alpha.list-title
+      %span.five.columns.alpha
+        = t "spree_admin_enterprises_hubs_name"
+      - if can? :admin, Spree::PaymentMethod
+        %span.centered.three.columns
+          = t(:payment_methods)
+      - if can? :admin, Spree::ShippingMethod
+        %span.centered.three.columns
+          = t "spree_admin_enterprises_shipping_methods"
+      - if can? :admin, EnterpriseFee
+        %span.centered.three.columns
+          = t "spree_admin_enterprises_fees"
+    %div.sixteen.columns.alpha.list
+      - @enterprises.is_distributor.each do |enterprise|
+        %a.sixteen.columns.alpha.list-item{ class: "#{cycle('odd','even')}", href: "#{main_app.edit_admin_enterprise_path(enterprise)}" }
+          %span.five.columns.alpha
+            = enterprise.name
+          %span.symbol.three.columns.centered
+            - if can? :admin, Spree::PaymentMethod
+              - payment_method_count = enterprise.payment_methods.count
+              - if payment_method_count > 0
+                %span.icon-ok-sign{ 'ofn-with-tip' => "#{pluralize payment_method_count, 'payment method'}" }
+              - else
+                %span.icon-remove-sign{ 'ofn-with-tip' => t('.has_no_payment_methods', enterprise: enterprise.name) }
+            - else
+              &nbsp;
+          %span.symbol.three.columns.centered
+            - if can? :admin, Spree::ShippingMethod
+              - shipping_method_count = enterprise.shipping_methods.count
+              - if shipping_method_count > 0
+                %span.icon-ok-sign{ 'ofn-with-tip' => "#{pluralize shipping_method_count, 'shipping method'}" }
+              - else
+                %span.icon-remove-sign{ 'ofn-with-tip' => t('.has_no_shipping_methods', enterprise: enterprise.name) }
+            - else
+              &nbsp;
+          %span.symbol.three.columns.centered
+            - if can? :admin, EnterpriseFee
+              - fee_count = enterprise.enterprise_fees.count
+              - if fee_count > 0
+                %span.icon-ok-sign{ 'ofn-with-tip' => "#{pluralize fee_count, 'fee'}" }
+              - else
+                %span.icon-warning-sign{ 'ofn-with-tip' => t('.has_no_enterprise_fees', enterprise: enterprise.name) }
+            - else
+              &nbsp;
+          %span.two.columns.omega.right
+            %span.icon-arrow-right
+
+    %a.sixteen.columns.alpha.button.bottom.blue{ href: "#{main_app.admin_enterprises_path}" }
+      = t "spree_admin_overview_enterprises_footer"
+      %span.icon-arrow-right
```

Comment:
I'm finding this a bit hard to follow without actually looking at it in the view, but it looks as though we're now not rendering any producers in the dashboard? I guess that is fine, just wanted to make sure that intention was to remove the completely.

Reply:
Oops that's not he idea. I'll check again.